packages/trader/src/Assets/Trading/Categories/trade-categories.tsx

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
```

Comment:
This file includes text description for unsupported in `DTrader` contracts (like `stay, end, asian, reset, callputspread `ant etc.), but existing in `SmartTrader`. Should we leave this text description for future when(if) these contracts will be transferred to `DTrater`? Or I can delete?

Reply:
yes, please keep